Libraries/PushNotificationIOS/RCTPushNotificationManager.mm

```diff
@@ -25,7 +25,7 @@
 
 static NSString *const kErrorUnableToRequestPermissions = @"E_UNABLE_TO_REQUEST_PERMISSIONS";
 
-#if !TARGET_OS_TV
+#if !TARGET_OS_UIKITFORMAC
```

Comment:
I'm not sure if this check should be removed or absence of the flag was is a mistake, because comment behind `endif` suggested that `TARGET_OS_UIKITFORMAC` flag should be present in the check, but it wasn't.